.travis.yml

```diff
@@ -3,7 +3,10 @@ language: ruby
 env:
   matrix:
   - "CANTALOUPE_VERSION=stable"
+  - "CANTALOUPE_VERSION=stable_with_delegate"
```

Comment:
Couldn't we just test for the presence of the DELEGATE_URL in the ENV? Why do we need a different version value? You can matrix values like:

- "CANTALOUPE_VERSION=stable" "DELEGATE_URL=http://a.url.here"
- "CANTALOUPE_VERSION=stable"
- "CANTALOUPE_VERSION=dev" "DELEGATE_URL=http://a.url.here"
- "CANTALOUPE_VERSION=dev"

Reply:
The environment variable doesn't actually pass through, which is a natural assumption, I understand why you'd make it. We have to re-create the Docker ENV in the cantalupe_spec.rb file (scan on down a bit).

Reply:
I should pass through. Travis matrices are just system ENVs. See below.